test/tiled_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:async';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:flame/flame.dart';
+import 'package:flutter/services.dart' show CachingAssetBundle;
+import 'package:test/test.dart';
+import '../lib/tiled_component.dart';
+
+void main() {
+  test('my first widget test', () async {
```

Comment:
`my first widget test`?

Reply:
O copied this from the original repository :P

But I am gonna change this